packages/openneuro-components/src/search-page/SearchResultItem.tsx

```diff
@@ -144,7 +153,7 @@ export const SearchResultItem = ({ node, profile }: SearchResultItemProps) => {
           : "Participants' Ages"}
         :{' '}
       </strong>
-      <span>{ages(node?.metadata?.ages)}</span>
+      <span>{ages(latestSnapshotSummary.subjectMetadata?.age)}</span>
```

Comment:
```suggestion
      <span>{ages(latestSnapshotSummary.subjectMetadata?.map(subject => subject.age)}</span>
```

subjectMetadata is an array of objects, not an object containing an array.